commands/channel.go

```diff
@@ -358,7 +358,7 @@ func makeChannelPrivateCmdF(c client.Client, cmd *cobra.Command, args []string)
 		return errors.New("you can only change the type of public channels")
 	}
 
-	if _, response := c.ConvertChannelToPrivate(channel.Id); response.Error != nil {
```

Comment:
Why are we making this change?

Reply:
We used to have an endpoint to make a channel private (and the `ConvertChannelToPrivate` was making use of it) and then we implemented the `UpdateChannelPrivacy` one, which allows to change privacy in both ways. First one is not deprecated yet, but we rather use the new one to allow for deprecation

Reply:
Ok!